modules/ROOT/pages/license-key.adoc

```diff
@@ -1,55 +1,203 @@
 = License key
 :description_short: {productname} License Configuration Guide | {productname}
-:description: Learn how to configure license key and stop {productname} from running in the evaluation mode.
+:description: Learn how to configure a license key in {productname}.
 :keywords: {productname}, cloud, script, textarea, apiKey, faq, license key, frequently asked questions,
 
+== Overview
+{productname} 8 uses an enhanced license key system to ensure compliance with {productname} licensing terms.
+
 [IMPORTANT]
 ====
-{productname} 7 is licensed under the GNU General Public License Version 2 or later. A new configuration option called 'license_key' requires developers to make a conscious decision to use {productname} with the GPLv2+ license or with a commercial license.
+{productname} is licensed under the GNU General Public License Version 2 or later. A configuration option called `+'license_key'+` introduced in {productname} 7 requires developers to make a conscious decision to use {productname} with the GPLv2+ license or with a commercial license.
 
-If you are using {productname} in a self-hosted environment, a console log warning message will display if the license key config option is missing or invalid. This message aims to ensure compliance with licensing requirements and provide transparency during the evaluation period.
+When using {productname} 8 in a self-hosted environment, a license key must be provided and it must be valid. Otherwise, the editor will be set to readonly. A license key is not required when loading {productname} from {cloudname}, as it is already under a commercial license.
```

Comment:
I would prefer you said that a license key would be automatically issued and is not required to be set manually.